step_definitions/addPupilsToTheCourse-steps.js

```diff
@@ -6,20 +6,20 @@ const teacherLogin = require('../page-objects/teacherLogin');
 const createCourse = require('../page-objects/createCourse');
 const Login = require('../shared-objects/loginData');
 
-Given(/^teacher arrives on the Schul-Cloud page$/, function() {
+Given(/^teacher arrives on the Schul-Cloud page$/, function () {
 	return helpers.loadPage(Login.url, 10);
 });
-Given(/^teacher is logged in successfully$/, function() {
-	return teacherLogin.performLogin(Login.defaultTeacherUsername,Login.defaultTeacherpassword);
+Given(/^teacher is logged in successfully$/, function () {
+	return teacherLogin.performLogin(Login.defaultTeacherUsername, Login.defaultTeacherpassword);
 });
 
-Given('teacher goes to courses page', function() {
+Given('teacher goes to courses page', function () {
 	return createCourse.goToCourses();
 });
```

Comment:
Why do we have to redefine these steps in every test case? Probably not a problem with your PR, but the repo in general

Reply:
I'm not sure..

Reply:
Technically it doesn’t matter how you name your step definition files, or which step definitions you put in a file. You could have one giant file containing all your step definitions. However, as the project grows, the file can become messy and hard to maintain. Instead, it is recommended creating a separate *_steps.js file for each domain concept(=feature)

Reply:
What I mean is, can we import steps that are used in many test cases?

Reply:
This branch has been closed due to name change. See new pr
https://github.com/schul-cloud/integration-tests/pull/88